src/d3d10/meson.build

```diff
@@ -25,3 +24,10 @@ d3d10_core_dll = shared_library('d3d10core'+dll_ext, d3d10_core_src, d3d10_core_
 d3d10_core_dep = declare_dependency(
   link_with           : [ d3d10_core_dll ],
 )
+
+if platform != 'windows'
+  pkg.generate(d3d10_core_dll,
+    filebase: dxvk_pkg_prefix + 'd3d10core',
```

Comment:
If the prefix was `dxvk_` rather than `dxvk-`, I think we could omit `filebase` here completely, and just let Meson deduce it from the `shared_library`.